backfillers/marginfi/syncMarginFiBanks.ts

```
@@ -164,98 +179,150 @@ class MarginFiSyncService {
+            } else {
+              console.log(
+                `Bank for ${tokenSymbol} is not active or conversion failed`
+              );
+            }
+          } else {
+            console.log(`No bank found for token: ${tokenSymbol}`);
           }
         } catch (error) {
-          console.warn(`Failed to parse bank account ${account.pubkey.toString()}:`, error);
+          console.error(`Error fetching bank for ${tokenSymbol}:`, error);
+          continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)